nifi-registry/nifi-registry-core/nifi-registry-web-api/src/test/resources/keys/README.md

```diff
@@ -72,20 +72,10 @@ WD="/tmp/test-keys-$(date +"%Y%m%d-%H%M%S")"
 mkdir "$WD"
 cd "$WD"
 
-# copy existing CA key/cert pair to working directory, rename to default tls-toolkit names
+# copy existing CA key/cert pair to working directory, rename to default names
 cp /path/to/nifi-registry/nifi-registry-core/nifi-registry-web-api/src/test/resources/keys/ca-key.pem ./nifi-key.key
 cp /path/to/nifi-registry/nifi-registry-core/nifi-registry-web-api/src/test/resources/keys/ca-cert.pem ./nifi-cert.pem
 
```

Comment:
Is some sort of blurb needed here to replace the `tls-toolkit` invocation?

Reply:
Some alternative wording here and elsewhere in the docs would seem sensible, e.g. based on the recent [dev mailing list thread](https://lists.apache.org/thread/fny5phr1mq2pqo86hqs1lmsnfrqt06sf) started by @exceptionfactory

I think the important thing is to strike the right balance between pointing administrators at the need for obtaining certificates, but without detailing the multitude of possible ways to do so (as people's needs are often different for different environments)

Do we note a set of minimum requirements and suggest people look to obtain a certificate (in the required format) from a reputable CA, or maybe generate self-signed certificates for development deployments?

Reply:
We could point them at this walkthrough (link to target documentation?):
https://github.com/apache/nifi/blob/main/nifi-docs/src/main/asciidoc/walkthroughs.adoc#securing-nifi-with-mtls

... which will be homed here:
https://nifi.apache.org/docs/nifi-docs/html/walkthroughs.html#securing-nifi-with-mtls

Something like:
```
place new keystores / truststores (for more info, see: https://github.com/apache/nifi/blob/main/nifi-docs/src/main/asciidoc/walkthroughs.adoc#securing-nifi-with-mtls)
```

Reply:
Sounds reasonable, I couldn't recall whether I'd seen the new although being added to the docs already, but such a link seems fair

Reply:
Added...

Reply:
To be in keeping with other links within the NiFi docs, I'd suggest using Markdown link notation, e.g. `for more info, see the [walkthrough for securing NiFi with mTLS](https://nifi.apache.org/docs/nifi-docs/html/walkthroughs.html#securing-nifi-with-mtls)`